tests/functional/externalpages/pocket/test_navigation.py

```diff
@@ -0,0 +1,37 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import pytest
+from selenium.webdriver.common.by import By
+
+from pages.externalpages.pocket.about import AboutPage
+
+
+@pytest.mark.smoke
```

Comment:
I'm not entirely clear on what the supported browsers are supposed to be for Pocket, but when viewed in IE11 the mobile menu button isn't visible smaller viewport sizes? This might be a bug (?), but I think we can expect this test to fail because of it. If we don't care about IE for these pages, then you can remove the `smoke` marker here.

<img width="400" alt="Screenshot 2022-01-14 at 13 54 13" src="https://user-images.githubusercontent.com/400117/149526745-787e76d6-9a2e-489d-b0e2-6744538ab955.png">

Reply:
removed, will open a new issue to make the menu IE11 compatible pending answer in pocket-marketing slack

Reply:
update: we do support IE but will not include this test as it works on browserstack and is not critical enough for further selenium debugging